### PR TITLE
Add documentation for microBees integration

### PR DESCRIPTION
**Pull request by marcolettieri**
## Proposed change

Add Documentation for microBees core Integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/99573
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5225
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by joostlek** [comment]
Please open a PR for the brands repository.

File: source/_integrations/microbees.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: microBees
+description: Instructions on how to integrate microBees devices into Home Assistant.
+ha_category:
+  - Switch
+ha_release: 0.1.0
```
Comment (joostlek) [inline source/_integrations/microbees.markdown]:
```suggestion
ha_release: 2024.3
```

File: source/_integrations/microbees.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ha_codeowners:
+  - '@microBeesTech'
+ha_config_flow: true
+ha_domain: microbees
+ha_iot_class: Cloud Polling
+ha_platforms:
+  - light
```
Comment (joostlek) [inline source/_integrations/microbees.markdown]:
```suggestion
```

File: source/_integrations/microbees.markdown [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - switch
+ha_integration_type: integration
+---
+
+The `microbees` integration allows you to control your [microBees devices](https://www.microbees.com/) such as plugs, wall switches and bulbs.
```
Comment (joostlek) [inline source/_integrations/microbees.markdown]:
```suggestion
The microbees integration allows you to control your [microBees devices](https://www.microbees.com/) such as plugs and wall switches.
```

File: source/_integrations/microbees.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+To use this integration you need OAuth2 Client ID and Client Secret and your user credentials.
+
+To retrieve the OAuth2 Client ID and Client Secret go to [microBees Developer Dashboard](https://developers.microbees.com/dashboard), login with your microBees account and [create a new app](https://developers.microbees.com/dashboard/?p=wizard), choose a Label, select WebApplication and input https://my.home-assistant.io as Website URL.
```
Comment (joostlek) [inline source/_integrations/microbees.markdown]: What's a "Label"?

**Review by joostlek — DISMISSED** [review]

File: source/_integrations/microbees.markdown (hunk 4 above)
Comment (marcolettieri) [reply source/_integrations/microbees.markdown]: A label for the Oauth Client

**Comment by marcolettieri** [comment]
> Please open a PR for the brands repository.

@joostlek 
Here the PR for brand, was also accepted but closed
https://github.com/home-assistant/brands/pull/4645

**Comment by joostlek** [comment]
That brands PR is adding brands for a custom integration, but they should be added to the core integrations. Please open a new one and correct it

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Keep this PR limited to the platforms created in your initial core PR, you can re-add all platforms you added later via a separate PR.

./Klaas

File: source/_integrations/microbees.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: microBees
+description: Instructions on how to integrate microBees devices into Home Assistant.
+ha_category:
+  - Switch
+  - Sensor
+
```
Comment (klaasnicolaas) [inline source/_integrations/microbees.markdown]:
```suggestion
```

File: source/_integrations/microbees.markdown [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ha_release: 2024.3
+ha_codeowners:
+  - '@microBeesTech'
+ha_config_flow: true
+ha_domain: microbees
+ha_iot_class: Cloud Polling
+ha_platforms:
+  - switch
+  - sensor
+
```
Comment (klaasnicolaas) [inline source/_integrations/microbees.markdown]:
```suggestion
```

File: source/_integrations/microbees.markdown [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+The microbees integration allows you to control your [microBees devices](https://www.microbees.com/) such as plugs and wall switches.
+To use this integration you need OAuth2 Client ID and Client Secret and your user credentials.
+
+To retrieve the OAuth2 Client ID and Client Secret go to [microBees Developer Dashboard](https://developers.microbees.com/dashboard), login with your microBees account and [create a new app](https://developers.microbees.com/dashboard/?p=wizard), choose a Label for your App, select WebApplication and input https://my.home-assistant.io as Website URL.
+
+There is currently support for the following device types within Home Assistant:
+
+- **Switch**
+- **Sensor**
+
```
Comment (klaasnicolaas) [inline source/_integrations/microbees.markdown]:
```suggestion
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
The sensor platform was also added in a later PR and therefore does not belong in this one.

Instead of just naming the supported platform, you could also describe what a user can do with, for example, the switch entity.

File: source/_integrations/microbees.markdown [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+ha_release: 2024.3
+ha_codeowners:
+  - '@microBeesTech'
+ha_config_flow: true
+ha_domain: microbees
+ha_iot_class: Cloud Polling
+ha_platforms:
+  - switch
+
```
Comment (klaasnicolaas) [inline source/_integrations/microbees.markdown]:
```suggestion
```

File: source/_integrations/microbees.markdown [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+title: microBees
+description: Instructions on how to integrate microBees devices into Home Assistant.
+ha_category:
+  - Switch
+
```
Comment (klaasnicolaas) [inline source/_integrations/microbees.markdown]:
```suggestion
```

File: source/_integrations/microbees.markdown [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+The microbees integration allows you to control your [microBees devices](https://www.microbees.com/) such as plugs and wall switches.
+To use this integration you need OAuth2 Client ID and Client Secret and your user credentials.
+
+To retrieve the OAuth2 Client ID and Client Secret go to [microBees Developer Dashboard](https://developers.microbees.com/dashboard), login with your microBees account and [create a new app](https://developers.microbees.com/dashboard/?p=wizard), choose a Label for your App, select WebApplication and input https://my.home-assistant.io as Website URL.
+
+There is currently support for the following device types within Home Assistant:
+
+- **Switch**
+
```
Comment (klaasnicolaas) [inline source/_integrations/microbees.markdown]:
```suggestion
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Remove also the extra blank lines

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**